src/lib/plugins/live-cell/live-cell.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, OnInit, SimpleChange, ChangeDetectionStrategy } from '@angular/core';
+import { TemplateHostService } from '../../template/template-host.service';
+import { TdCoreDirective } from 'lib/main/core/body/td-core.directive';
+
+@Component({
+	selector: 'q-grid-live-cell',
+	templateUrl: './live-cell.tpl.html',
+	providers: [TemplateHostService],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LiveCellComponent implements OnInit {
+
+	@Input() cell: TdCoreDirective;
+	@Input() duration = 500;
+
+	difference = 0;
+
+	constructor() {
+	}
+
+	ngOnInit() {
+		if (!this.cell.changes) {
+			this.cell.mode('view');
+			return;
+		}
+		this.difference = this.diff(this.cell.changes);
```

Comment:
I think we need to propagate to live template `changes` not `difference`, for instance for text template difference doesn't make any sense.

Reply:
Also getDifference(changes) method can be added that calculates the value, not persist.